consensus/src/consensus_observer/observer.rs

```diff
@@ -1091,14 +1181,29 @@ fn sync_to_commit_decision(
     let (abort_handle, abort_registration) = AbortHandle::new_pair();
     tokio::spawn(Abortable::new(
         async move {
-            execution_client
+            // Sync to the commit decision
+            if let Err(error) = execution_client
```

Comment:
how does state sync handle the overwrited request? does it return to the previous request?

Reply:
Good question. If we issue two calls, the second one will overwrite the first and the first will just be dropped, so it should never return from the `await`, here. Still need to think if there's the potential for multiple sync requests to be issued at once 🤔

Reply:
Are you thinking there's the potential for an issue, here?

Reply:
if the request is dropped, the previous sync_to_target would return something like oneshot channel dropped? I don't think it'll cause issue but it probably shouldn't be an error log

Reply:
Aah, gotcha 😄 Let me downgrade these!